math/fastor/include/algebra/math/fastor.hpp

```diff
@@ -0,0 +1,14 @@
+/** Algebra plugins library, part of the ACTS project
+ *
+ * (c) 2020-2022 CERN for the benefit of the ACTS project
```

Comment:
Just a nitpicky detail, but the date can be 2022 outright

Reply:
Thanks for pointing that out. I've made the change, but I'll push it a bit later.